share/spice/crunchbase/collections/crunchbase_collections.js

```diff
@@ -0,0 +1,66 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_crunchbase_collections = function(api_result){
+        // throw error if no result, error result, no items, or array of 0 items
+        if (!api_result || api_result.error || !DDG.getProperty(api_result, 'data.items') || api_result.data.items.length === 0) {
+            return Spice.failed('crunchbase');
+        }
+        var data = api_result.data, // get org data
+	    items = data.items, // get all orgs
+	    firstResult = items[0], // get first org to render for now
+	    permalink = firstResult.path; // get endpoint of first org
+
+	var metadata = api_result.metadata; // get metadata for base urls
+	// get base urls if metdata exists
+	if (metadata){
+            var wwwPathPrefix = metadata.www_path_prefix,
+                imagePathPrefix = metadata.image_path_prefix;
+	}
+
+        // get organization information from organization endpoint
+	$.getJSON('js/spice/crunchbase/organizations/'+ permalink.split('/')[1],function(response) {
+            // return error if no response to organization query
+            if (!response) {
+                return Spice.failed('crunchbase');
+            } else {
```

Comment:
I think we can ditch these `else` blocks. It should help keep the code from being so heavily nested